platform/ios/PolyPodApp/PolyPod/en.lproj/Localizable.strings

```diff
@@ -44,6 +44,10 @@
 
 "auth_prompt_unlock" = "Login to your polyPod";
 
+"homescreen_footer_title" = "Like what you have seen?";
```

Comment:
Right, and this is something we'll have to replicate too using whatever Kotlin uses for this and so on and so forth.

Reply:
Correct.